I updated the patch, but it still fails to build with:
```
error: builder for '/nix/store/npv6skwpsmn6zmfjb6x5f673zf7laxp9-obs-ndi-4.13.0.drv' failed with exit code 2;
       last 10 log lines:
       >       |          ^~~~~~
       > compilation terminated.
       > make[2]: *** [CMakeFiles/obs-ndi.dir/build.make:76: CMakeFiles/obs-ndi.dir/src/plugin-main.cpp.o] Error 1
       > /build/source/src/obs-ndi-source.cpp:29:10: fatal error: QString: No such file or directory
       >    29 | #include <QString>
       >       |          ^~~~~~~~~
       > compilation terminated.
       > make[2]: *** [CMakeFiles/obs-ndi.dir/build.make:90: CMakeFiles/obs-ndi.dir/src/obs-ndi-source.cpp.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:111: CMakeFiles/obs-ndi.dir/all] Error 2
       > make: *** [Makefile:156: all] Error 2
       For full logs, run 'nix log /nix/store/npv6skwpsmn6zmfjb6x5f673zf7laxp9-obs-ndi-4.13.0.
```

Not sure yet what dependency is missing for `QString`, let me know

--- a/pkgs/applications/video/obs-studio/plugins/obs-ndi/default.nix
+++ b/pkgs/applications/video/obs-studio/plugins/obs-ndi/default.nix
@@ -4,7 +4,7 @@ stdenv.mkDerivation rec {
   pname = "obs-ndi";
   version = "4.13.0";
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake qtbase ];
   buildInputs = [ obs-studio qtbase ndi ];
 
   src = fetchFromGitHub {
@@ -28,12 +28,7 @@ stdenv.mkDerivation rec {
     ln -s ${ndi}/include lib/ndi
   '';
 
-  postInstall = ''
-    mkdir $out/lib $out/share
-    mv $out/obs-plugins/64bit $out/lib/obs-plugins
-    rm -rf $out/obs-plugins
-    mv $out/data $out/share/obs
-  '';
+  cmakeFlags = [ "-DENABLE_QT=ON" ];
 
   dontWrapQtApps = true;
 

--- a/pkgs/applications/video/obs-studio/plugins/obs-ndi/hardcode-ndi-path.patch
+++ b/pkgs/applications/video/obs-studio/plugins/obs-ndi/hardcode-ndi-path.patch
@@ -10,7 +10,7 @@ index 0d94add..617af73 100644
 -	locations << "/usr/lib";
 -	locations << "/usr/local/lib";
 -#endif
-+	locations << "@NDI@/lib"
++	locations << "@NDI@/lib";
  	for (QString location : locations) {
  		path = QDir::cleanPath(
  			QDir(location).absoluteFilePath(NDILIB_LIBRARY_NAME));

